tests/hardforks.ts

```diff
@@ -90,8 +90,8 @@ tape('[Common]: Hardfork logic', function(t: tape.Test) {
     c = new Common('ropsten', null, ['spuriousDragon', 'byzantium', 'constantinople'])
     st.equal(
       c.activeHardforks(null, { onlySupported: true }).length,
-      2,
-      'should return 2 active HFs when restricted to supported HFs',
+      3,
+      'should return 3 active HFs when restricted to supported HFs',
```

Comment:
We could add another test here that checks that `activeHardforks` behaves correctly when `Common` is instantiated with at least one non-supported hardfork and `activeHardforks` is called with the `onlySupported` option. This would fulfill the original purpose of the test. Could use `dao` as done below with one of the `hardforkGteHardfork` tests.

example:
```
    c = new Common('ropsten', null, ['spuriousDragon', 'byzantium', 'dao'])
    st.equal(
      c.activeHardforks(null, { onlySupported: true }).length,
      3,
      'should return 3 active HFs when restricted to supported HFs',
    )
```

Reply:
Have added the additional test (result of this should be 2 though, to match the two supported from the three restricted to).

Thanks for the review! 😄 Will directly merge on tests passing since this was the only change requested and changes are not production-code touching.